components/OgImage/OgImageNuxter.vue

```diff
@@ -40,7 +40,7 @@ const format = useNumberFormatter()
           </svg>
           <div class="font-medium text-4xl pl-2 pt-1">{{ format(contributor.score) }}</div>
         </div>
-        <div class="flex flex-row text-gray-400 items-center pt-3 font-medium text-3xl">
+        <div class="flex flex-row text-[#94a3b8] items-center pt-3 font-medium text-3xl">
```

Comment:
what's the reason for not using tailwind colours?

Reply:
Alright, I updated my branch from upstream and it seems like the tailwind colors issue has been resolved with some dependency updates so I reverted back the classes from my PR